src/main/java/org/example/homework_nr_5/HomeWork5.java

```diff
@@ -0,0 +1,62 @@
+package org.example.homework_nr_5;
+enum Month{
+    JANUARY, FEBRUARY, MARCH, APRIL, MAY, JUNE, JULY, AUGUST, SEPTEMBER, OCTOBER, NOVEMBER, DECEMBER
+}
+enum weekday{
```

Comment:
This works, but it's very inefficient. Remove the trues and falses from the enums and use the this keyword in the methods below.

Something like:
`public boolean isWeekDay(){
if(this == SUNDAY || this == SATURDAY) return false; else return true;`